glib/src/commonMain/kotlin/io/github/cgr/glib/impls/IoOps.kt

```diff
@@ -0,0 +1,21 @@
+@file:JvmName("IoOps")
```

Comment:
1. I'm curious as to why this is necessary
2. It feels incredibly cursed that you're using it *before* you're importing it

Reply:
1. Because if you try calling this from Java the methods will be in `IoOpsKt`, not `IoOps`
2. Yeah, `file` annotations have to be that way